app/models/guide.rb

```diff
@@ -13,4 +13,20 @@ class Guide < ActiveRecord::Base
   before_validation on: :create do |object|
     object.content_id = SecureRandom.uuid
   end
+
+  def work_in_progress_edition?
+    latest_edition.try(:state) == 'draft'
+  end
+
+  def update_attributes_from_params(params, state:)
```

Comment:
No, that was not my reasoning. It's a good point though, but we should use Active Model validations on an edition model to solve this. That's a separate task imho